vizro-core/src/vizro/static/css/aggrid.css

```diff
@@ -0,0 +1,46 @@
+#page-container .ag-theme-alpine.vizro {
+  --ag-background-color: var(--main-container-bg-color);
+  --ag-foreground-color: var(--text-primary);
+  --ag-odd-row-background-color: var(--main-container-bg-color);
+  --ag-header-foreground-color: var(--text-primary);
+  --ag-header-background-color: var(--main-container-bg-color);
+  --ag-header-column-resize-handle-display: none;
```

Comment:
Removing this makes it harder for the user to resize the columns. Is there some other solution that would be consistent with the design?  Maybe show on hover?

Reply:
Hm good point - I have been thinking the same. I will speak to our designer about this, but for the time being let's leave them removed. The functionality is still there if one desires to resize columns - and it can be overwritten of course if it is very important 💪